Google.PowerShell.IntegrationTests/Container2/Container.GkeNodePool.Tests.ps1

```diff
@@ -30,6 +30,31 @@ $clusterThreeCreationJob = Start-Job -ScriptBlock $clusterCreationScriptBlock `
 
 Wait-Job $clusterOneCreationJob, $clusterTwoCreationJob, $clusterThreeCreationJob | Remove-Job
 
+# Check that a cluster is running.
+function Check-Cluster($clusterName, $clusterZone) {
+    $cluster = Get-GkeCluster -ClusterName $clusterName -Zone $clusterZone
+    return $cluster.Status -eq "Running"
+}
+
+$clusterNames = @($clusterOneName, $clusterTwoName, $clusterThreeName)
+$clusterZones = @($zone, $clusterTwoZone, $clusterThreeZone)
+
+# Wait until every cluster is fully started. Wait for at most 10 minutes.
```

Comment:
The creation is done in parallel so we will most likely just need to wait for the first one.